udata/core/metrics/__init__.py

```diff
@@ -111,10 +112,12 @@ def iso(self, value):
                 'Unsupported format: {0} ({1})'.format(value, type(value)))
 
     def trigger_update(self):
-        self.need_update.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
```

Comment:
Yep, it should be documented, but I prefer a `USE_METRICS` or `ENABLE_METRICS` instead of force metrics (see next comment)